lib/omniauth/strategies/openid_connect.rb

```diff
@@ -258,10 +259,14 @@ def user_info
 
         if access_token.id_token
           decoded = decode_id_token(access_token.id_token).raw_attributes
+          merge_with = JSON::JWS.new({})
```

Comment:
As far as I can tell, `::OpenIDConnect::ResponseObject::UserInfo.new` works on a regular Hash; what's the deal with using `JSON::JWS`?